tests/run/cpp_stl_algo_sample.pyx

```diff
@@ -0,0 +1,39 @@
+# mode: run
+# tags: cpp, cpp17
+
+from libcpp.algorithm cimport sample
+from libcpp.iterator cimport back_inserter
+from libcpp.random cimport mt19937
+from libcpp.utility cimport move
+from libcpp.vector cimport vector
+
+
+def sample_multiple(population_size, int sample_size):
+    """
+    >>> sample = sample_multiple(10, 7)
+    >>> len(sample), len(set(sample))  # Check sampling without replacement.
+    (7, 7)
+    """
+    cdef:
+        vector[int] x, y
+        int i
+        mt19937 rd = mt19937(1)
+
+    [x.push_back(i) for i in range(population_size)]
```

Comment:
Throwing away the result of a list comprehension is a bit of a funny spelling for a for-loop. I'd rather see the for-loop to avoid making readers think about the effect of this.